makefile

```diff
@@ -30,5 +27,5 @@ docker-build:
 docker-up:
 	docker-compose up -d
 
-start:
+dev:
```

Comment:
docker compose is not the best solution for local development - it is hard to troubleshoot - too long feedback. It does not connected to current PR, but it could be improved!